content/en/functions/collections/D.md

```diff
@@ -0,0 +1,49 @@
+---
+title: collections.D
+description: Returns a slice of sequentially ordered random integers.
+categories: []
+keywords: [random]
+params:
+  functions_and_methods:
+    returnType: '[]int'
+    signatures: [collections.D SEED N MAX]
+---
+
+{{< new-in 0.149.0 />}}
+
+The `collections.D` function returns a slice of `N` sequentially ordered unique random integers in the half-open [interval](g) [0, `MAX`) using the provided `SEED` value. This function implements J. S. Vitter's Method&nbsp;D[^1] for sequential random sampling, a fast and efficient algorithm for this task.
+
+See [this article][] for a detailed explanation.
+
+```go-html-template
+{{ collections.D 6 7 42 }} → [4, 9, 10, 20, 22, 24, 41]
+```
+
+The example above generates the _same_ random numbers each time it is called. To generate a _different_ set of 7 random numbers in the same range, change the seed value.
+
+```go-html-template
+{{ collections.D 2 7 42 }} → [3, 11, 19, 25, 32, 33, 38]
+```
+
+> [!note]
+> All arguments are cast to integers, so setting the seed to `3.14` is the same as setting it to `3`.
+
+A common use case is the selection of random pages from a page collection. For example, to render an unordered list of 5 random pages using the current time as the seed value:
```

Comment:
It's an ordered list (which is a difference from shuffle, but a difference I suspect most people prefer).